server/grpc_service.go

```diff
@@ -222,33 +247,38 @@ type tsoRequest struct {
 	stream        pdpb.PD_TsoServer
 }
 
-func (s *GrpcServer) dispatchTSORequest(ctx context.Context, request *tsoRequest, forwardedHost string, doneCh <-chan struct{}, errCh chan<- error) {
+func (s *GrpcServer) dispatchTSORequest(ctx context.Context, request *tsoRequest, forwardedHost string, doneCh <-chan struct{}, errCh chan<- error, workingMCS bool) {
```

Comment:
How about using a clearer name instead of `workingMCS`?

Reply:
Do you have a good idea?

Reply:
The difference inside of dispatchTSORequest is actually about which protocolbuf to use, pdpb or tsopb? How about 'tsoProto bool'. The tso service itself needs to use this framework too.

Reply:
ok